### PR TITLE
feat: add example for lazy

### PR DESCRIPTION
**Pull request by chunhuigao**

### CODE REVIEW

File: pages/lazy/index.vue [hunk 1]
```diff
@@ -0,0 +1,52 @@
+<template>
+  <div class="example">
+    <app-type> Basic Usage </app-type>
+    <var-space>
+      <img class="cat" v-lazy="'https://varlet.gitee.io/varlet-ui/cat.jpg'" />
```
Comment (qytayh) [inline pages/lazy/index.vue:5]: why not use v-lazy="lazy"

File: pages/lazy/index.vue [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+<script lang="ts">
+import { defineNuxtComponent } from '#app'
+import AppType from '~/components/appType.vue'
+export default defineNuxtComponent({
+  components: {
+    AppType,
+  },
+  setup() {
+    const data = reactive({
+      lazy: 'https://varlet.gitee.io/varlet-ui/cat.jpg',
```
Comment (qytayh) [inline pages/lazy/index.vue:36]: you may use ref

Comment (chunhuigao) [reply pages/lazy/index.vue:36]: copy

Comment (chunhuigao) [reply pages/lazy/index.vue:36]: > you may use ref

copy

**Pull request closed without merging**